documentation/database/testcontainers.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: documentation
+menu: testcontainers
+subtitle: TestContainers
+---
+# TestContainers
+
+## Driver
+<table class="table">
+<tr>
+<th>URL format</th>
+<td><code>jdbc:tc:</code> instead of <code>jdbc:</code> for your database</td>
+</tr>
+<tr>
+<th>Ships with Flyway Command-line</th>
+<td>Yes</td>
+</tr>
+<tr>
+<th>Maven Central coordinates</th>
+<td><code>org.testcontainers:jdbc:jar:1.14.3</code></td>
```

Comment:
we should probably list all the jars required, so the final ones (postgres, mysql, etc) and all the other dependencies